js_modules/dagster-ui/packages/ui-core/src/ui/VirtualizedTable.tsx

```diff
@@ -5,18 +5,32 @@ import styled from 'styled-components';
 export const HeaderCell = ({
   children,
   style,
+  onClick,
 }: {
   children?: React.ReactNode;
   style?: React.CSSProperties;
-}) => (
-  <CellBox
-    padding={{vertical: 8, horizontal: 12}}
-    border="right"
-    style={{whiteSpace: 'nowrap', textOverflow: 'ellipsis', overflow: 'hidden', ...(style || {})}}
-  >
-    {children}
-  </CellBox>
-);
+  onClick?: React.MouseEventHandler;
```

Comment:
Rather than just passing through `onClick`, I would be more in favor of just doing 

```
export const HeaderCell = ({
  children,
  style,
  onClick,
}: Omit<React.ComponentProps<typeof CellBox>, 'padding' | 'border'>) => {
  ...
}
```

Though maybe we should allow overriding the padding/border too.